lib/jemoji.rb

```diff
@@ -8,8 +8,16 @@ class Emoji
 
     class << self
       def emojify(doc)
+        return unless doc.output =~ HTML::Pipeline::EmojiFilter.emoji_pattern
         src = emoji_src(doc.site.config)
-        doc.output = filter_with_emoji(src).call(doc.output)[:output].to_s
+        if doc.output =~ /<body/
+          parsed_doc = Nokogiri::HTML::Document.parse(doc.output)
+          body       = parsed_doc.at_css('body')
+          body.replace filter_with_emoji(src).call(body.to_html)[:output]
```

Comment:
Let's remove this `.to_html` – from what I could see in the docs, it can operate on the fragment, too.